ksp_plugin.py

```diff
@@ -204,11 +205,14 @@ def run(self):
                 else:
                     log_message('Compiling...')
 
-                self.compiler = ksp_compiler.KSPCompiler(code, self.base_path, compact, compact_variables,
-                                                         read_file_func=self.read_file_function,
-                                                         extra_syntax_checks=check,
-                                                         optimize=optimize and check,
-                                                         add_compiled_date_comment=add_compiled_date_comment)
+                self.compiler = ksp_compiler.KSPCompiler(code, self.base_path, 
+                                                        compact             = compact,
+                                                        compact_variables   = compact_variables,
+                                                        extra_syntax_checks = check,
+                                                        combine_callbacks   = combine_callbacks,
+                                                        read_file_func=self.read_file_function,
```

Comment:
Let's align all those arguments to the equals sign, yeah?

For that matter, add one more whitespace to the indent so that it aligns to the first argument, `code`.